LICENSES/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
The file name probably needs to be LICENSES/Apache-2.0.txt, see for example here: https://github.com/SAP/fosstars-rating-core/blob/master/LICENSES/Apache-2.0.txt Please validate locally using the REUSE tool.